cross-platform/react-app/src/frontend/BottomPanels/ToolsBottomPanel.tsx

```diff
@@ -110,7 +110,7 @@ export function getDefaultTools(): ToolEntry[] {
 
 export function ToolsBottomPanel(props: ToolsBottomPanelProps) {
   const defaultTools = React.useMemo(getDefaultTools, []);
-  const { iModel, onToolClick, tools = defaultTools, ...others } = props;
+  const { iModel: _iModel, onToolClick, tools = defaultTools, ...others } = props;
```

Comment:
Wouldn't it be better to simply remove the iModel extraction from props?

Reply:
I couldn't decide which way to go here. The `others` object gets passed onto BottomPanel and iModel isn't an expected/used prop. If you think I should remove it, that's fine with me.

Reply:
Ah, sorry. I hadn't noticed that bit. I think that what you have is the best solution.